sconscontrib/SCons/Tool/pandoc/__init__.py

```diff
@@ -529,9 +529,12 @@ def exists(env):
             f"Could not parse panflute version {panflute_version_}"
         )
 
-    if any(pandoc_version < (2, 10) and panflute_version >= (2,),
-           pandoc_version > (2, 10) and panflute_version < (2,)):
-        raise SCons.Errors.StopError(
+    import panflute
```

Comment:
I'd suggest the `import panflute` should also be wrapped in a try block since not installed is a potential outcome

Reply:
Agree. Otherwise you'll throw an exception if it's not installed at all.

Reply:
The version check near [line 515](https://github.com/SCons/scons-contrib/blob/master/sconscontrib/SCons/Tool/pandoc/__init__.py#L515) is supposed to catch an invalid panflute installation.  If panflute is installed but isn't reporting a valid version, I'd say it wasn't properly installed.

Reply:
Gotcha! Didn't see that in the github diff of your PR until I fully expand it. Thanks!